numpyro/distributions/constraints.py

```diff
@@ -73,11 +73,20 @@ def check(self, value):
         """
         return self(value)
 
+    def _default_value(self, shape):
+        """
+        Get default value with shape specified in `shape`.
+        """
```

Comment:
Update api

Reply:
Thanks for the catch! I don't have preference. :D